AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Device.java

```diff
@@ -159,29 +147,24 @@ public int getPhoneID() {
         return mPhoneID;
     }
 
+    private String getSimInformation(String simInfo, String errorMsg) {
+        try {
+            if(simInfo != null && !simInfo.isEmpty())
```

Comment:
Before your changes, the `tm.getSim...()` methods were called inside of `try`-blocks, but after your changes the methods are called outside of the try block, so the block is pretty useless